drivers/lora/sx12xx_common.c

```diff
@@ -148,5 +148,13 @@ int sx12xx_init(const struct device *dev)
 	dev_data.events.RxDone = sx12xx_ev_rx_done;
 	Radio.Init(&dev_data.events);
 
+	/* Automatically place the radio into sleep mode upon boot.
```

Comment:
Nit. Please start the comment from next line.

Reply:
This seems to be a matter of personal preference. On other PR's I have been asked to use this form :). A quick regex search on this form (`\/\*[^\r\*\/]+$`) brings up over 4000 occurrences.

Reply:
I will update this usage as the other occurrence in this file uses the newline form